WordPress/Helpers/RulesetPropertyHelper.php

```diff
@@ -43,15 +43,24 @@ final class RulesetPropertyHelper {
 	 * @since 2.0.0  No longer supports custom array properties which were incorrectly
 	 *               passed as a string.
 	 * @since 3.0.0  Moved from the Sniff class to this class.
+	 * @since 3.1.0  Added a new parameter to lowercase array keys and values.
```

Comment:
Should this be `3.1.0` or `3.0.2` ? It's a new feature, true, but it is on an internal class which doesn't have any BC promise and the actual patch is about _fixing a bug_.